### PR TITLE
Support application account in runtime, update checkpoint structure

### PR DESCRIPTION
**Pull request by ratik21**
## Proposed Changes

+ Support `applicationAccount` in runtime
+ Add key `applicationAccount` to app's checkpoints

### CODE REVIEW

File: packages/algob/src/lib/algo-operator.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { types as rtypes } from "@algo-builder/runtime";
 import { BuilderError, ERRORS, tx as webTx, types as wtypes } from "@algo-builder/web";
-import algosdk, { LogicSigAccount, modelsv2 } from "algosdk";
+import algosdk, { getApplicationAddress, LogicSigAccount, modelsv2 } from "algosdk";
```
Comment (robert-zaremba) [inline packages/algob/src/lib/algo-operator.ts:3]: Can we rename to `getAppAddress`?

Comment (robert-zaremba) [reply packages/algob/src/lib/algo-operator.ts:3]: Where is this defined? Probably some file is missing in this PR?

Comment (ratik21) [reply packages/algob/src/lib/algo-operator.ts:3]: it is defined in the js-sdk: [https://github.com/algorand/js-algorand-sdk/blob/develop/src/encoding/address.ts#L187](https://github.com/algorand/js-algorand-sdk/blob/develop/src/encoding/address.ts#L187)

Comment (robert-zaremba) [reply packages/algob/src/lib/algo-operator.ts:3]: ah, sorry, didn't look at the end of the statement ;)

**Review by robert-zaremba — APPROVED** [review]

**Pull request merged**